packages/verify/src/timestamp/index.ts

```diff
@@ -49,45 +41,12 @@ export function verifyTSATimestamp(
   };
 }
 
-export function verifyTLogTimestamp(
-  entry: TransparencyLogEntry,
-  tlogAuthorities: TLogAuthority[]
+export function getTLogTimestamp(
+  entry: TransparencyLogEntry
 ): TimestampVerificationResult {
-  let inclusionVerified = false;
-
-  if (isTLogEntryWithInclusionPromise(entry)) {
-    verifyTLogSET(entry, tlogAuthorities);
-    inclusionVerified = true;
-  }
-
-  if (isTLogEntryWithInclusionProof(entry)) {
-    verifyMerkleInclusion(entry);
-    verifyCheckpoint(entry, tlogAuthorities);
-    inclusionVerified = true;
-  }
-
-  if (!inclusionVerified) {
-    throw new VerificationError({
-      code: 'TLOG_MISSING_INCLUSION_ERROR',
-      message: 'inclusion could not be verified',
-    });
-  }
-
```

Comment:
There verification of the tlog entry is now happening later in the verification workflow. At this point, we're just extracting the `integratedTime` from the TLog entry (if it's available). The verification will happen later.